addons/mail/static/tests/message/message_tests.js

```diff
@@ -1594,3 +1594,28 @@ QUnit.test("Message should display attachments in order", async () => {
     await contains(":nth-child(2 of .o-mail-AttachmentCard)", { text: "B.txt" });
     await contains(":nth-child(3 of .o-mail-AttachmentCard)", { text: "C.txt" });
 });
+
+QUnit.test("Can edit a message with an empty body", async () => {
```

Comment:
I would rename test "Can edit a message only containing an attachment".

It feels weird to say "edit a message with an empty body" by itself, because it lacks context for making relevant for such messages to be shown in UI. This is especially true when empty messages should not be shown and definitely not being editable.

Reply:
Done :)